core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -136,7 +144,7 @@
     </div>
 
     <dot-edit-ema-palette
-        *ngIf="es.isEnterpriseLicense && !currentDevice && es.editor.page.canEdit"
+        *ngIf="es.isEnterpriseLicense && !currentDevice && !seoMedia && es.editor.page.canEdit"
```

Comment:
This is totally getting out of hand.

Reply:
I'll make a refactor to unify the logic in one object.

Reply:
I'll do something better, I'll move this to the store.